openpgp/integration_tests/utils.go

```diff
@@ -0,0 +1,216 @@
+package integrationtests
```

Comment:
Similarly, I think this file should end in "_test.go", even though it doesn't make much sense, so that `go build` doesn't include this package.

Reply:
+1